spec/Section 2 -- Language.md

```diff
@@ -166,7 +167,7 @@ Ignored ::
   - Comma
 
 {Ignored} tokens are used to improve readability and provide separation between
-{Token}, but are otherwise insignificant and not referenced in syntactical
+{Token}s, but are otherwise insignificant and not referenced in syntactical
```

Comment:
I think it will look strange in rendered spec.
Can you change the phrase without adding `s`?